recipes/binutils/all/conanfile.py

```diff
@@ -206,6 +206,8 @@ def package_info(self):
         # Don't use this property in production code. It's unsupported.
         self.user_info.recipe_path = os.path.realpath(__file__)
 
```

Comment:
```suggestion
        self.cpp_info.resdirs = ["etc"]
        self.cpp_info.buildenv_info.define("GPROFNG_SYSCONFDIR", os.path.join(self.package_folder, "etc"))
```
The configuration file is not found so easily without this.